appdaemontestframework/hass_mocks.py

```diff
@@ -94,6 +94,7 @@ def _hass_init_mock(self, _ad, name, *_args):
             MockHandler(Hass, 'turn_on'),
             MockHandler(Hass, 'turn_off'),
             MockHandler(Hass, 'fire_event'),
+            MockHandler(Hass, 'select_option'),
```

Comment:
Could you please add a test for it here: https://github.com/FlorianKempenich/Appdaemon-Test-Framework/blob/master/test/test_extra_hass_functions.py